src/Aspire.Hosting.Keycloak/KeycloakResourceBuilderExtensions.cs

```diff
@@ -152,9 +152,10 @@ public static IResourceBuilder<KeycloakResource> WithDataBindMount(this IResourc
         ArgumentNullException.ThrowIfNull(builder);
         ArgumentNullException.ThrowIfNull(importDirectory);
 
-        if (!Directory.Exists(importDirectory))
+        var importDirectoryFullPath = Path.GetFullPath(importDirectory, builder.ApplicationBuilder.AppHostDirectory);
+        if (!Directory.Exists(importDirectoryFullPath))
         {
-            throw new DirectoryNotFoundException($"The realm import directory {importDirectory} does not exist.");
+            throw new DirectoryNotFoundException($"The realm import directory {importDirectory} does not exist under {builder.ApplicationBuilder.AppHostDirectory} .");
         }
 
         builder.WithBindMount(importDirectory, RealmImportDirectory, isReadOnly);
```

Comment:
Would it make sense to change this:

```suggestion
        builder.WithBindMount(importDirectoryFullPath, RealmImportDirectory, isReadOnly);
```

That way we ensure the path we are checking above is the path being used.